### PR TITLE
issue_335 changes to tool labels and button positions

### PR DESCRIPTION
**Pull request by zqian**

### CODE REVIEW

**Comment by pushyamig** [comment]
I have updated this branch from latest from main

File: frontend/app/components/ToolCard.tsx [hunk 1]
```diff
@@ -119,6 +119,14 @@ export default function ToolCard (props: ToolCardProps) {
...
           aria-busy={updateToolNavLoading}
         >
+          <Button
+            onClick={() => setShowMoreInfo(!showMoreInfo)}
+            aria-expanded={showMoreInfo}
+            aria-label={`Show ${moreOrLessText} Info`}
+            startIcon={!showMoreInfo ? <ExpandMoreIcon /> : <ExpandLessIcon />}
+          >
+            {moreOrLessText}
+          </Button>
```
Comment (pushyamig) [inline frontend/app/components/ToolCard.tsx:129]: Minor detail: The Mock up now say `Learn more`  change reflect `More`. don't know if we are sticking with existing design from CAE as discussed in the Slack yesterday. Keeping the changes to minimal. I am ok if that's the case

**Review by pushyamig — APPROVED** [review]
Looks Good

File: frontend/app/components/ToolCard.tsx (hunk 1 above)
Comment (zqian) [reply frontend/app/components/ToolCard.tsx:129]: I will keep the "More" vs "Less" labels for now. Thanks!

Comment (jonespm) [reply frontend/app/components/ToolCard.tsx:129]: I'd have to look at the notes but I know we had a lot of UX back and forth about this button. I also believe it had an accessibility issue and I'm not sure if it was resolved. I'll have to see it when it's running, but that would be a separate issue.

**Review by jonespm — APPROVED** [review]

**Pull request merged**